designs/UI/user-profile.html

```diff
@@ -0,0 +1,165 @@
+                <li class="nav-item active">
+                  <a class="nav-link" href="#">WEECONNECT: User Profile View<span class="sr-only">(current)</span></a>
+                </li>
+                <!-- <li class="nav-item">
+                  <a class="nav-link" href="#">Link</a>
+                </li>
+                <li class="nav-item dropdown">
+                  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    Dropdown
+                  </a>
+                  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                    <a class="dropdown-item" href="#">Action</a>
+                    <a class="dropdown-item" href="#">Another action</a>
+                    <div class="dropdown-divider"></div>
+                    <a class="dropdown-item" href="#">Something else here</a>
+                  </div>
+                </li>
+                <li class="nav-item">
+
+                        <a class="nav-link disabled" href="#">Disabled</a>
+                    </li>
+                  -->
+                </ul> 
+                  <!-- <form class="form-inline my-2 my-lg-0">
+                    <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+                    <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+                  </form> -->
+                </div>
+              </nav>
+
+                    <!-- <form class="form-inline">
+                      <div class="input-group">
+                        <div class="input-group-prepend">
+                          <span class="input-group-text" id="basic-addon1">@</span>
+                        </div>
+                        <input type="text" class="form-control" placeholder="Username" aria-label="Username" aria-describedby="basic-addon1">
+                      </div>
+                    </form>
+                  </nav> -->
+
+
+                  <div class="container-fluid">
+                      <div class="row">
+                          <div class="col-xs-6 col-sm-4">
+                          </div>
+                          <div class="col-xs-6 col-sm-8"></div>
+                      </div>
+                  </div>
+                  <div class="container-fluid">
+                      <div class="row">
+                          <div class="col-xs-6 col-sm-4">
+                              <div class="card">
+                                  <div class="card-header deep-orange lighten-1 white-text">Personal Information</div>
+                                  <div class="card-body">
+
+                                  <form class="text-center">
+                                    
+                                    <img src="images/user.jpg" alt="User Image" style="width:150px; height:150px;" class="rounded-circle">
```

Comment:
Use external css instead of inline css

Reply:
ok, will try my best to use external css